backend/apps/cache/services/cache_manager.py

```
@@ -340,23 +340,73 @@ def wrapper(*args, **kwargs):
+    import sys
+    # Check if we're in a manage.py command (migration)
+    if 'manage.py' in sys.argv and 'migrate' in sys.argv:
+        return True
+    try:
+        from django.db import connection
+        with connection.cursor() as cursor:
+            # Check if contenttypes table has the name column properly
+            cursor.execute("PRAGMA table_info(django_content_type)")
+            columns = {row[1] for row in cursor.fetchall()}
+            return 'name' not in columns
+    except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)